index.js

```diff
@@ -1 +1,39 @@
-// JAVASCRIPT CODE HERE
\ No newline at end of file
+import BookCollection2 from './modules/BookCollection2.js';
+/* global luxon */
+const luxonDateTime = luxon.DateTime;
+
+const addBooks = document.getElementById('addBooks');
+const contact = document.getElementById('contact');
+const section2 = document.getElementById('section2');
+const section3 = document.getElementById('section3');
+const section4 = document.getElementById('section4');
+const heading = document.getElementById('heading');
+
+addBooks.addEventListener('click', (event) => {
+  event.preventDefault();
+  section2.style.display = 'none';
+  section3.style.display = 'block';
+  section4.style.display = 'none';
+  heading.style.display = 'none';
+});
+
+contact.addEventListener('click', (event) => {
+  event.preventDefault();
+  section2.style.display = 'none';
+  section3.style.display = 'none';
+  section4.style.display = 'block';
+  heading.style.display = 'none';
+});
+
+const bookCollection1 = new BookCollection2();
+bookCollection1.displayBooks();
+
+function updateDateTime() {
```

Comment:
* [ ] Your function works perfectly, and that's great! As this project specifically requires the use of ES6 syntax, I would recommend utilizing arrow functions instead of regular functions. This practice aligns with the ES6 standard and enhances the readability and maintainability of the code. Please consider updating your code to use arrow functions wherever appropriate. Keep up the good work, and happy coding! 🚀🎉